modules/@apostrophecms/image-widget/index.js

```diff
@@ -140,6 +137,8 @@ module.exports = {
     };
   },
   init(self) {
+    self.hidePlaceholder = self.options.placeholder === false;
+    self.options.placeholder = true;
```

Comment:
I use `hidePlaceholder` instead of placeholder that must stay true.

Reply:
I think that we have this tacit rule to always use positive variables.

I'd recommend using `showPlaceholder = self.options.placeholder !== false;`

Reply:
I'm ok with that!
Done!